Packages/com.unity.inputsystem/InputSystem/NativeInputRuntime.cs

```diff
@@ -394,22 +393,22 @@ public Action onProjectChange
 
         public void SendAnalytic(InputAnalytics.IInputAnalytic analytic)
         {
-            #if (UNITY_EDITOR)
+        #if (UNITY_EDITOR && ENABLE_CLOUD_SERVICES_ANALYTICS)
```

Comment:
Looks alright, sorry for overlooking this. But why not just add a single #if ENABLE_CLOUD_SERVICES_ANALYTICS outside the existing editor checks since all of it should be excluded?

Reply:
Thanks for the suggestion, improved it now based on what you said